projects/api/src/routes/auth.ts

```diff
@@ -26,5 +31,10 @@ export const authRouter: typeof Router = (): Router => {
         checkError(res, response, () => res.json(response))
     })
 
+    router.post('/onboarding', validateAuthentication(), validateBody(onboardingSchema), async (req: IAuthenticatedRequest, res: Response) => {
+        const response: object | Boom = await authController.onboarding(req.user!, req.body)
```

Comment:
`req.user` cannot be null or undefined? Ah because the middleware already check that

Reply:
Yes that's what I explained